frontend/src/components/MainToolbar.vue

```diff
@@ -28,13 +28,30 @@ SPDX-License-Identifier: Apache-2.0
             <span>Info</span>
           </v-tooltip>
         </template>
-        <v-card tile>
+        <v-card tile width="300px">
           <v-card-title primary-title>
             <div class="content">
               <div class="title mb-2">Gardener</div>
               <v-progress-circular size="18" indeterminate v-if="!dashboardVersion"></v-progress-circular>
               <div class="caption" v-if="!!gardenerVersion">API version {{gardenerVersion}}</div>
               <div class="caption" v-if="!!dashboardVersion">Dashboard version {{dashboardVersion}}</div>
+              <v-expansion-panels flat v-if="extensionCount && isAdmin">
```

Comment:
I think this info should not be displayed in the help menu.

Reply:
Yes, I agree. We can also hide this completely for the moment and make an additional PR where we add a decent about dialog. Alternatively, I can include i in this PR WDYT?

Reply:
removed extension list from this PR, will file an additional PR after this one is merged where we build an about screen to display this kind of information